runtime/src/commonTest/kotlin/pbandk/testpb/test.kt

```diff
@@ -29,6 +29,7 @@ data class Foo(
                 )
             }
             pbandk.MessageDescriptor(
+                fullName = "testpb.Foo",
```

Comment:
For consistency pbandk.testpb.Foo?

Reply:
`fullName` contains the name of the _protobuf_ message, not the name of the _Kotlin_ generated class. So this is `testpb` rather than `pbandk.testpb` because the `test.proto` file that this `test.kt` file was generated from contains a `package testpb` declaration.